slack-base/src/main/java/com/hubspot/slack/client/models/auth/OAuthV2CredentialsIF.java

```diff
@@ -0,0 +1,36 @@
+package com.hubspot.slack.client.models.auth;
+
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.teams.SlackTeamLite;
+import com.hubspot.slack.client.models.users.SlackUserLite;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public abstract class OAuthV2CredentialsIF {
+  public abstract String getAccessToken();
+
+  @JsonProperty("team")
+  public abstract SlackTeamLite getSlackTeamLite();
```

Comment:
I think this can be just `getTeam`. This way we don't need to override json property name.